### ISSUE TITLE
MessageDispatcher error on macOS

### ISSUE THREAD

**Issue by JonZeolla** [PowerShell/vscode-powershell#1173]
### System Details

- Operating system name and version:  macOS 10.13.2
- VS Code version: 1.19.2
- PowerShell extension version: 1.5.1
- Output from `$PSVersionTable` (and other commands):

```
> code -v
1.19.2
490ef761b76b3f3b3832eff7a588aac891e5fe80
x64
> $pseditor.EditorServicesVersion

Major  Minor  Build  Revision
-----  -----  -----  --------
1      5      1      0


> code --list-extensions --show-versions
ms-vscode.PowerShell@1.5.1
> $PSVersionTable
Name                           Value
----                           -----
PSVersion                      6.0.0-rc.2
PSEdition                      Core
GitCommitId                    v6.0.0-rc.2
OS                             Darwin 17.3.0 Darwin Kernel Version 17.3.0: Thu Nov  9 18:09:22 PST 2017; root:xnu-4570.31.3~1/RELEASE...
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Issue Description

I am seeing `DispatchMessage` errors in my vscode-powershell logs.  This may be related to `startSessionCommand` being removed (https://github.com/Microsoft/vscode/issues/33791) per [this](https://github.com/Microsoft/vscode/issues/35974) issue.

### Attached Logs

From `~/.vscode/extensions/ms-vscode.powershell-1.5.1/logs/1516906362-f4c99603-def9-4124-a625-19cd0fa620bd1516761624124/EditorServices.log`:
```
1/25/18 1:52:48 PM [ERROR] - Method "DispatchMessage" at line 147 of C:\projects\powershelleditorservices\src\PowerShellEditorServices.Protocol\MessageProtocol\MessageDispatcher.cs^M

    MessageDispatcher: No handler registered for Event type 'initialized'

1/25/18 1:52:48 PM [ERROR] - Method "DispatchMessage" at line 147 of C:\projects\powershelleditorservices\src\PowerShellEditorServices.Protocol\MessageProtocol\MessageDispatcher.cs^M

    MessageDispatcher: No handler registered for Event type '$/setTraceNotification'
```

**Comment by rkeithhill** [comment]
Those messages just indicate that the extension doesn't do anything with those message/event types.  The vscode language and debug protocols are tweaked over time in order to add new functionality that extensions can then choose to implement (or not).  

Usually not handling a message means you might be missing out on some functionality but it shouldn't cause an issue.  Well, except for maybe the fact that we don't handle `cancelRequest` msgs and while that doesn't cause an immediate issue there is concern that it might cause a perf issue.

**Comment by JonZeolla** [comment]
I was able to validate that those errors were not causing my primary issue (which has been addressed in powershell/powershelleditorservices#592 but not yet released).  I'll leave this ticket open if it's useful to you @rkeithhill.

**Comment by rkeithhill** [comment]
Thanks for the update.  We are tracking the `cancelRequest` issue on the PSES repo - https://github.com/PowerShell/PowerShellEditorServices/issues/602  So I'm going to close this.
